src/ui/components/Viewer.tsx

```diff
@@ -31,12 +31,10 @@ const Vertical = ({ toolboxLayout }: { toolboxLayout: ToolboxLayout }) => {
   useLayoutEffect(() => {
     const videoPanel = videoPanelRef.current;
     if (videoPanel) {
-      if (videoPanel.isCollapsed() !== videoPanelCollapsed) {
```

Comment:
This change seems fishy to me. I think it _should_ work, and the fact that it didn't suggests a bug in react-resizable-panels. Stepping through the code, it looks like the problem is that I don't initialize `collapsedSize` to 0 when it's undefined, so it fails in the library here:

```js
  const isPanelCollapsed = useCallback(panelData => {
    const {
      layout,
      panelDataArray
    } = eagerValuesRef.current;
    const {
      collapsedSize,
      collapsible,
      panelSize
    } = panelDataHelper(panelDataArray, panelData, layout);
    return collapsible === true && panelSize === collapsedSize;
  }, []);
```

I guess this change is probably also okay to make though either way. I'll file a follow up issue.

Reply:
Yeah, I didn't go all the way into the lib code, but I wondered if it might be something like that.

Given that this is a "sync external behavior" situation, always calling the relevant `collapse/expand` method seems like reasonable implementation behavior.

Reply:
Fixed upstream and updated our dependency in #10063